ArduCopter/switches.cpp

```diff
@@ -624,6 +625,38 @@ void Copter::do_aux_switch_function(int8_t ch_function, uint8_t ch_flag)
             }
 #endif
             break;
+
+        case AUXSW_WINCH_ENABLE:
+            switch (ch_flag) {
+                case AUX_SWITCH_HIGH:
+                    // high switch maintains current position
+                    g2.winch.release_length(0.0f);
+                    Log_Write_Event(DATA_WINCH_LENGTH_CONTROL);
+                    break;
+                default:
+                    // all other position disable winch
+                    g2.winch.relax();
+                    Log_Write_Event(DATA_WINCH_DISABLE);
```

Comment:
What is the difference supposed to be between disable/relax and maintaining current position?

Reply:
"relax" will allow the motor to move while maintaining the current position will cause the controller to increase/decrease the motor output to maintain the rope at the current position.